x/staking/keeper/validator.go

```diff
@@ -191,10 +182,7 @@ func (k Keeper) RemoveValidator(ctx sdk.Context, address sdk.ValAddress) {
 	store.Delete(types.GetValidatorsByPowerIndexKey(validator, k.PowerReduction(ctx)))
 
 	// call hooks
-	err = k.AfterValidatorRemoved(ctx, valConsAddr, validator.GetOperator())
```

Comment:
Why no longer catching this error?

Reply:
the hook only through an error in SDK 46 - in 45 it does not throw an error